### ISSUE TITLE
Some python driver versions don't work properly on Ubuntu 14.04 (throws a VersionConflict)

### ISSUE THREAD

**Issue by vrde** [rethinkdb/rethinkdb#5402]
I noticed this behavior installing our package [BigchainDB](https://github.com/bigchaindb/bigchaindb) an AWS instance with Ubuntu 14.04. Our package installs a couple of shell scripts, and they both fail to run because of a `VersionConflict` exception. If you need, our [`setup.py` is available in our repo](https://github.com/bigchaindb/bigchaindb/blob/develop/setup.py) as well as the issue https://github.com/bigchaindb/bigchaindb/issues/41.

This issue affects all the packages where:
 - the value of [`rethinkdb.version.version`](https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/version.py) is different than the value of `Version` in `PKG-INFO` in the [source distribution](https://pypi.python.org/packages/source/r/rethinkdb/rethinkdb-2.2.0.post3.tar.gz), and
 - the system is using the default version of `setuptools` shipped with Ubuntu 14.04 (that is, `setuptools==3.3`). The last version of `setuptools` works properly, but I think the problem is on how the driver is packaged on your side.

### Get a shell in ubuntu 14.04

```sh
docker run --rm -it ubuntu:14.04
```
### Once you are inside the container
```sh
apt-get install python3-setuptools -y
easy_install3 pip
pip install rethinkdb==2.2.0.post3
python3
```

### Once you are in the interpreter
```python
>>> import pkg_resources as p
>>> ws = p.WorkingSet()
>>> ws.resolve([p.Requirement.parse('rethinkdb==2.2.0.post3')])

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/pkg_resources.py", line 632, in resolve
    raise VersionConflict(dist,req) # XXX put more info here
pkg_resources.VersionConflict: (rethinkdb 2.2.0-3 (/usr/local/lib/python3.4/dist-packages), Requirement.parse('rethinkdb==2.2.0.post3'))
```

## But, it works in this case
In the container, try to install an alternative version of the driver

```sh
pip install rethinkdb==2.1.0
```

### Once you are in the interpreter
```python
>>> import pkg_resources as r
>>> 
>>> ws = r.WorkingSet()
>>> 
>>> ws.resolve([r.Requirement.parse('rethinkdb==2.1.0')])
[rethinkdb 2.1.0 (/usr/local/lib/python3.4/dist-packages)]    # <-- yee, no exception!
```

## Conclusion
I think that the problem is when the package name ends with `postX`, like `rethinkdb 2.2.0.post3`, `rethinkdb 2.2.0.post2`, `rethinkdb 2.2.0.post1` and on. I inspected the packages in [pypi](https://pypi.python.org/pypi/rethinkdb) and there is a discrepancy between the version in the `rethinkdb.version` package and the version contained in `PKG-INFO`.

**Comment by danielmewes** [comment]
Thanks for the (greatly written) issue report.

At some point Pypi started to automatically translate our versions to the `.postX` format. We should probably just adapt that version format for `rethinkdb.version` as well, so both versions become consistent again.

I'll push a new release of the latest Python driver with the changed version format later, and post here once it's up.

**Comment by danielmewes** [comment]
Just published 2.2.0.post4 https://pypi.python.org/pypi/rethinkdb/2.2.0.post4 .
@vrde Can you check if that one works correctly?

**Comment by vrde** [comment]
> @vrde Can you check if that one works correctly?

@danielmewes thanks for the prompt reply and fix, it works perfectly.
